### PR TITLE
[Hotfix] Fix Zoom Lerp/Sensitivity settings not being there

### PR DESCRIPTION
**Pull request by BraedonWooding**
This isn't really a hotfix more of an addition of an omission.  You could still control these settings through adding them by hand in the settings file (located under appdata/locallow for windows and library somewhere for mac).  

This just adds them as sliders in settings menu which was literally just a few lines of xml no custom logic needed which was nice.  

I also fixed a localization bug that affected generic sliders due to them doing their localization slightly wrong (which resulted in it having localized text like my_localized_description instead of a nice format like My Localized Description or whatever).  Just adds an extra parameter to create text to control whether or not to localize the passed in text (since in reality it shouldn't if you are localizing part cause else it'll have problems).  Anyways it works  so yah.

This also removes the Settings.xml and Settings.json file that was just sitting in the project not used, while it may be nice since it reminds people of structure it often leads to confusion of people thinking that's the file that is read rather than the one in appData, and it's incorrect.  Though if enough people want it (for some reason I might be missing) I'll add it back in an updated one.

### CODE REVIEW

File: listOfFiles [hunk 1]
```diff
@@ -0,0 +1,1412 @@
+.gitattributes
```
Comment (bjubes) [inline listOfFiles]: I assume you accidentally added this?

Comment (BraedonWooding) [inline listOfFiles]: I must have, this is weird xD.  I'll remove it

**Comment by BraedonWooding** [comment]
@koosemose you okay with me removing the project files settings.json and settings.xml since we never use them in code and use the one under `AppData/LocalLow` and the project ones just cause confusion?

**Comment by koosemose** [comment]
Yeah... I think at one point the original (.xml) was used for something, but when it was changed to not use it, it was never removed, so it continued getting updated by people assuming it must be used for something.

**Pull request merged**